hazelcast-spring/src/main/resources/hazelcast-spring-4.1.xsd

```diff
@@ -1573,8 +1573,9 @@
                     <xs:element name="user-code-deployment" type="user-code-deployment-client" minOccurs="0" maxOccurs="1"/>
                     <xs:element name="flake-id-generator" type="client-flake-id-generator" minOccurs="0" maxOccurs="unbounded"/>
                     <xs:element name="reliable-topic" type="client-reliable-topic" minOccurs="0" maxOccurs="unbounded"/>
-                    <xs:element name="metrics" type="client-metrics" minOccurs="0" maxOccurs="unbounded"/>
+                    <xs:element name="metrics" type="client-metrics" minOccurs="0" maxOccurs="1"/>
```

Comment:
We should have only one `metrics` element in the config. This is not really effective though, since we have `<choice maxOccurs="unbounded">` as the enclosing element.